### PR TITLE
Prepare Blueno-UI for deployment on app engine

### PR DESCRIPTION
**Pull request by luke-zhu**
Notable changes:

- We likely don't need Backend.js since the proxy seems to be working
- I'm suggesting we use the credentials in google app engine from now on

### CODE REVIEW

**Comment by therladbsgh** [comment]
- Not sure why we would want to move server files to root. If it's a requirement from Google App Engine then we don't really have a choice, but why would we have a separate folder for front-end (src/) but not for the back-end (+ it messes up some files, but that's not too big of a deal)

- The purpose of Backend.js is more specifically to provide helper function for fetch commands, especially for POST and file uploads. Since we have the proxy variable we don't need BASE_URI though.

- Using dotenv seems like a good idea for standardization

File: main.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    root_logger.setLevel(logging.DEBUG)
+
+    handler = logging.StreamHandler()
+    formatter = logging.Formatter(
+        fmt='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+    handler.setFormatter(formatter)
+    root_logger.addHandler(handler)
+
+
+@app.route('/', defaults={'path': ''})
```
Comment (therladbsgh) [inline main.py]: Throws an error, since index.html no longer exists

File: routes/preprocess.py [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+
+def __process_file(files, user, date, bucket):
+    client = pymongo.MongoClient(
+        "mongodb://bluenoml:elvoanalysis@104.196.51.205/elvo"
+    )
+    db = client.elvo.datasets
+    for file in files:
+        # try:
+        npy = preprocess.process_cab_from_file(file, file.filename,
+                                               '../tmp/cab_files/')
```
Comment (therladbsgh) [inline routes/preprocess.py]: Need to fix. Now throws an error since it should now be tmp/cab_files

File: src/utils/Backend.js [hunk 3]
```diff
@@ -1,10 +1,21 @@
 // Should be changed as appropriate
-export const BASE_URI = 'http://localhost:8080';
-// const BASE_URI = 'http://ec2-34-229-105-203.compute-1.amazonaws.com:5000/';
+
+// TODO(luke): This won't work with HTTPS
```
Comment (therladbsgh) [inline src/utils/Backend.js]: We don't need this anymore

**Comment by luke-zhu** [comment]
Yeah since the entrypoint is main.py, I was only able to get the paths/module imports to make sense with main at the top level. This shouldn't be a big issue since the recommendation is to move to this pattern: http://flask.pocoo.org/docs/1.0/patterns/packages/ anyway

**Comment by luke-zhu** [comment]
This looks good, we'll just need to use `gsutil` to standardize the elvo buckets. I think at some point we should look at getting more of the user work integrated.

After looking more into what our data architecture would look like in the future, I'm a big fan of moving towards a managed SQL service (probably Google Cloud Postgres). It seems like we'll have to do some wierd things to make sure that MongoDB satisfies [hipaa](https://www.mongodb.com/blog/post/making-hipaa-compliant-applications-mongodb)

**Comment by luke-zhu** [comment]
Let's meet up today to continue the standardization. I also want to hear what features you find important as well.

**Pull request merged**